env-example

```diff
@@ -0,0 +1,11 @@
+ASPNETCORE_ENVIRONMENT=Production
+ASPNETCORE_URLS=https://0.0.0.0:443/
+ASPNETCORE_Kestrel__Certificates__Default__Password=[your password]
+ASPNETCORE_Kestrel__Certificates__Default__Path=/app/ssl.pfx
+DbConnectionString=""
```

Comment:
According to [docker env file docs](https://docs.docker.com/compose/env-file/), quotation marks are not handled and **are part of the ENV file**.